python/paddle/fluid/layers/nn.py

```diff
@@ -9081,12 +9081,9 @@ def _attr_offsets_check(offset_val):
     return out
 
 
-def affine_grid(theta, out_shape, name=None):
+def affine_grid(theta, out_shape, name=None, align_corners=True,
```

Comment:
put `name` last, remove `use_cudnn`

a new version should be be add in nn.functional since arguments have changed